scripts/create-clusters.sh

```diff
@@ -113,10 +114,12 @@ function create-clusters() {
     unset KUBECONFIG
   fi
 
-  # TODO(font): Configure insecure registry on kind host cluster. Remove once
-  # https://github.com/kubernetes-sigs/kind/issues/110 is resolved.
-  echo "Configuring insecure container registry on kind host cluster"
-  configure-insecure-registry-on-cluster 1
+  if [[ "${CONFIGURE_INSECURE_REGISTRY_HOST}" ]]; then
```

Comment:
This one should be `CONFIGURE_INSECURE_REGISTRY_CLUSTER` and the original `CONFIGURE_INSECURE_REGISTRY` can be renamed to `CONFIGURE_INSECURE_REGISTRY_HOST`.

Reply:
Done